apps/site/lib/site_web/templates/customer_support/_accessibility.html.eex

```diff
@@ -0,0 +1,8 @@
+<p>
+  To submit an ADA/Accessibility complaint or feedback, please call the number listed above (or repeat call center details) or complete the feedback form to the left. The Customer Engagement Coordinator System-Wide Accessibility & Customer Liaison for The RIDE are responsible for coordinating the resolutions of ADA/Accessibility complaints once they are received.
```

Comment:
> (or repeat call center details)

This doesn't seem right to stay in the final copy - are you sure this is correct?

![image](https://user-images.githubusercontent.com/2136286/230415427-ca67a5e6-d67c-4673-92d0-22869113f611.png)

Reply:
That was the approved copy. Will remove for now and await feedback